src/Node.hpp

```diff
@@ -82,6 +83,11 @@ class Node
   Publisher<T> create_publisher(CanardPortID const port_id,
                                 CanardMicrosecond const tx_timeout_usec);
 
+  template <typename T>
+  Subscription<T> create_subscription(CanardPortID const port_id,
```

Comment:
Would it not be more consistent to say "subscriber" here? That, or replace "publisher" with "publication".

Reply:
I felt the same but the ROS2 API also distinguishes between a `Publisher` and a `Subscription`, see [here](https://docs.ros.org/en/foxy/Tutorials/Beginner-Client-Libraries/Writing-A-Simple-Cpp-Publisher-And-Subscriber.html#write-the-subscriber-node). After thinking on it I'd say this distinction is justified, as the `Publisher` instantiation is used again after creation via `publish` while there's really no need to use the `Subscription` instance anymore, except keeping it on memory somewhere. Ergo `Subscription` is much more passive and really not more than that, a subscription.